MonkeyLoader/Components/ComponentNotFoundException.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+
+namespace MonkeyLoader.Components
+{
+    /// <summary>
+    /// Represents the error when no matching component could be found in an
+    /// <see cref="IComponentList{TEntity}.Get{TComponent}()"/> or
+    /// <see cref="IComponentList{TEntity}.Get{TComponent}(Predicate{TComponent})"/> method.
+    /// </summary>
+    public sealed class ComponentNotFoundException : Exception
+    {
+        /// <summary>
+        /// Creates a new instance of this exception.
+        /// </summary>
+        public ComponentNotFoundException() : base("No component of matching type found!")
+        { }
+    }
+}
```

Comment:
Should this maybe contain _what_ was trying to be looked for that wasn't found..? Seems like generic enough to not be useful at all without any info of the source without proper handling...